gtsam_unstable/slam/tests/testTSAMFactors.cpp

```diff
@@ -121,18 +129,26 @@ TEST( OdometryFactorBase, all ) {
 
   // Use numerical derivatives to calculate the Jacobians
   Matrix H1Expected, H2Expected, H3Expected, H4Expected;
+  // using lambdas to replace bind
   H1Expected = numericalDerivative11<Vector3, Pose2>(
-      std::bind(&OdometryFactorBase::evaluateError, &factor, std::placeholders::_1, pose1, base2,
-          pose2, boost::none, boost::none, boost::none, boost::none), base1);
+      [&factor, &pose1, &pose2, &base2](const Pose2& pose_arg) {
+        return factor.evaluateError(pose_arg, pose1, base2, pose2);
+      },
+      base1);
```

Comment:
FYI for future reference I think it works if we do `boost::none` -> `(OptionalMatrixType*) 0` (or static_cast), but lambda looks better anyway :)

Reply:
Ah I see